.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
If you do it with = sign in the app or from the keyboard

![image](https://github.com/user-attachments/assets/6872e2b9-4d36-42af-8f9d-c7f028ce8556)

But if you do it with Enter key from keyboard it results in 0 in both lines, when in normal operations it works with Enter

![image](https://github.com/user-attachments/assets/2bebc502-2960-48de-9cd6-d9797374bb5b)